core/nativeaot/HelloWorld/Dockerfile

```diff
@@ -8,9 +8,9 @@ RUN apt-get update \
 WORKDIR /source
 
 COPY . .
-RUN dotnet publish -c release -r linux-x64 -o /app
+RUN dotnet publish -o /app
```

Comment:
Users will get a warning here if they open this project with Dev Kit because it will create a solution file. Best practice now is to add the project file to this line to avoid that.